src/roswire/ros2/launch.py

```diff
@@ -39,26 +39,7 @@ def read(self,
              package: Optional[str] = None,
              argv: Optional[Sequence[str]] = None
              ) -> LaunchConfig:
-        """Produces a summary of the effects of a launch file.
-
-        Parameters
-        ----------
-        filename: str
-            The name of the launch file, or an absolute path to the launch
-            file inside the container.
-        package: str, optional
-            The name of the package to which the launch file belongs.
-        argv: Sequence[str], optional
-            An optional sequence of command-line arguments that should be
-            supplied to :code:`roslaunch`.
-
-        Raises
-        ------
-        PackageNotFound
-            If the given package could not be found.
-        LaunchFileNotFound
-            If the given launch file could not be found in the package.
-        """
+        # TODO Implement this by constructing a launch file reader
```

Comment:
It looks like we're missing an implementation here.

Reply:
Also, we should keep the original docstring.

Reply:
Added docstring back. I was thinking that the implementation would be part of the